cmd/vocli/commands/keys-eth.go

```diff
@@ -0,0 +1,52 @@
+package commands
```

Comment:
What is the reasoning for having different files for keys?. I think given that this file have 3 functions, it can be merged into keys.go file.
But I do not strongly oppose to this approach :)

Reply:
Functions in this file are the shim between us and the Ethereum libraries. Keeps things neat.